### ISSUE TITLE
how to use the evaluation server？

### ISSUE THREAD

**Issue by Chenzixi1** [SilvioGiancola/TrackingNet-devkit#22]
Thanks for your work.

I wonder how to use evalation server. 

In [evalution](http://eval.tracking-net.org/web/challenges/challenge-page/39/evaluation), there is no way to eval but the link to this github, I am very confused.

Or, can we get the GT of TestDataset?

**Comment by SilvioGiancola** [comment]
Hi @Chenzixi1,

We segregated the ground truth for the test set for participant to avoid overfitting on it, so you won't have access to it. Once you have trained a model (train/val as you like) you can infer your results and send them to our evaluation server.

See [here](https://github.com/SilvioGiancola/TrackingNet-devkit#evaluate-the-results-of-a-tracker-with-a-given-ground-truth) for details on how to evaluate your predictions, in particular [metrics.py](https://github.com/SilvioGiancola/TrackingNet-devkit/blob/master/metrics.py) that provides you with the evaluation function currently used in EvalAI. You can zip any inference results and with any GT to evaluate its performances on any list of tracking sequence.
